docs/_includes/partials/service-worker.njk

```diff
@@ -1,8 +1,9 @@
+{% if site.url !== '/_site-dev/' %}
 <script>
   if ('serviceWorker' in navigator) {
     window.addEventListener('load', function() {
       navigator.serviceWorker
-        .register(`${window.location.pathname.split('/').length - 1 > 1 ? '../service-worker.js' : './service-worker.js' }`)
+        .register('service-worker.js')
```

Comment:
wont this break?

Reply:
In what way? The PR build seems to work as expected:
https://5eb7085fc83fc00007895dc0--open-wc-org.netlify.app/index.html

![image](https://user-images.githubusercontent.com/1156657/81516213-21d5ca00-9305-11ea-811f-ea2a9fb4ea90.png)

You've got a lot more face time with service worker development, so I'll trust you if you think there's something amiss. In theory, the `<base>` tag points this to the right place all the time...

Reply:
Similarly, even if we didn't use a `<base>` tag, that _could_ be achieved by `/service-worker.js` all the time if we prevent the service worker while in development?

Reply:
o.O if `open-wc.org/linting/linting-eslint.html` registers `service-worker.js` i would expect it to look for a service worker in `/linting/`, which is not there because its at the root `/`, right?

Would be nice if we dont need the window.location hack, im just surprised to see that's working and trying to understand why 😅

Reply:
yes pls no `<base>` tag and using `/service-worker.js` 👍
we do not have a service worker for dev anyways (as we don't run rollup for this)